drivers/clock_control/clock_stm32_ll_u3.c

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <soc.h>
+#include <stm32_ll_bus.h>
+#include <stm32_ll_pwr.h>
+#include <stm32_ll_rcc.h>
+#include <stm32_ll_utils.h>
+#include <stm32_ll_system.h>
+#include <zephyr/arch/cpu.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+
+
+/* Macros to fill up prescaler values */
+#define z_ahb_prescaler(v) LL_RCC_HCLK_SYSCLK_DIV_ ## v
+#define ahb_prescaler(v) z_ahb_prescaler(v)
+
+#define z_apb1_prescaler(v) LL_RCC_APB1_HCLK_DIV_ ## v
+#define apb1_prescaler(v) z_apb1_prescaler(v)
+
+#define z_apb2_prescaler(v) LL_RCC_APB2_HCLK_DIV_ ## v
+#define apb2_prescaler(v) z_apb2_prescaler(v)
+
+#define z_apb3_prescaler(v) LL_RCC_APB3_HCLK_DIV_ ## v
+#define apb3_prescaler(v) z_apb3_prescaler(v)
+
+static uint32_t get_bus_clock(uint32_t clock, uint32_t prescaler)
+{
+	return clock / prescaler;
+}
+
+static uint32_t get_msis_frequency(void)
+{
+	uint32_t frequency = MSIRC1_VALUE;
+
+	if (LL_RCC_MSIS_GetClockSource() == LL_RCC_MSIS_CLOCK_SOURCE_RC0)
+	{
+	frequency = MSIRC0_VALUE;
+	}
```

Comment:
Indentation + braces
```suggestion
	if (LL_RCC_MSIS_GetClockSource() == LL_RCC_MSIS_CLOCK_SOURCE_RC0) {
		frequency = MSIRC0_VALUE;
	}
```